prow/config.yaml

```diff
@@ -96,6 +96,9 @@ branch-protection:
             teams:
             - stage-bots
         dashboard:
+          required_status_checks:
+            contexts:
+            - continuous-integration/travis-ci
```

Comment:
Consider migrating the job to prow

Reply:
I am not really familiar with it, how much work is needed to do so?

Reply:
it depends on what you're doing, see [here](https://github.com/kubernetes/test-infra/blob/master/prow/pod-utilities.md#writing-a-prowjob-that-uses-pod-utilities), it can be fairly simple to get a command running in a container, but depending on what your test does obviously things can get more complicated to move around.

Reply:
@nikhita might also be able to lend a hand :-)

Reply:
> @nikhita might also be able to lend a hand :-)

Somehow just saw this now. :woman_facepalming: 

@maciaszczykm I can take a look later this week. :)

Reply:
This missed my radar, and I am not sure if I have enough bandwidth to work on prow migration until early next week so please don't block on me. :grimacing: